### PR TITLE
Upgrade Jollyday to 0.27.0

### PR DESCRIPTION
**Pull request by wborn**
Upgrades Jollyday from 0.23.2 to 0.27.0.

For release notes, see:

https://github.com/focus-shift/jollyday/releases

### CODE REVIEW

File: itests/org.openhab.core.model.rule.tests/itest.bndrun [hunk 1]
```diff
@@ -126,10 +124,11 @@ Fragment-Host: org.openhab.core.model.rule.runtime
 	org.objectweb.asm.tree;version='[9.6.0,9.6.1)',\
 	org.objectweb.asm.tree.analysis;version='[9.6.0,9.6.1)',\
 	org.objectweb.asm.util;version='[9.6.0,9.6.1)',\
-	org.openhab.core.model.persistence.runtime;version='[4.2.0,4.2.1)',\
```
Comment (kaikreuzer) [inline itests/org.openhab.core.model.rule.tests/itest.bndrun:129]: This removal seems unrelated. Any idea why it is there and if it is ok to be removed?

Comment (wborn) [reply itests/org.openhab.core.model.rule.tests/itest.bndrun:129]: I just let the resolver resolve the bundles and it came up with these bundles. The tests still run fine without it. There is probably some implementation detail that makes it resolve different bundles depending on the local environment. Maybe it's due to the OS, Java version or the order of files on disk. To workaround it we could blacklist those bundles that continuously get added/removed by the resolver.

Comment (kaikreuzer) [reply itests/org.openhab.core.model.rule.tests/itest.bndrun:129]: Ok, thanks!

**Review by kaikreuzer — APPROVED** [review]
